### ISSUE TITLE
Power Fist's Pickaxe, Axe and Shovel modules not working

### ISSUE THREAD

**Issue by Xydru** [MachineMuse/MachineMusePowersuits#954]
Hey,

 

I am playing my modpack in 1.7.10 and I recently crafted a power fist. I installed the pickaxe, axe and shovel modules as well as a battery that is completely charged. I overclocked the modules to harvest speed x30, but there wasn't any effect. It was as fast as a normal stone tool. All the other modules seemed to work fine.

I didn't set any keybinds, so that can't be the problem and I am in gamemode 0.

I play on a server, but I also tested it in singleplayer and it didn't work.

I installed the newest version.

I tried out cobblestone, cleanstone, oak wood, oak planks, dirt and sand

 

I would very appreciate it if somebody could help me.

**Comment by Diresheep** [comment]
did you install a battery in any piece of power armor/power fist?

**Comment by Xydru** [comment]
> did you install a battery in any piece of power armor/power fist?

Yes, I also tried different batteries and they were completely charged. I only installed one in the power fist, because I don't own some armor of MPS yet
